contracts/SampleToken.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.4.18;
+
+import "zeppelin-solidity/contracts/token/ERC20/MintableToken.sol";
+import { Jurisdiction } from "./tpl-contracts/Jurisdiction.sol";
+
+/**
+ * @title SampleToken
+ * @dev Mintable ERC20 Token.
+ */
+contract SampleToken is MintableToken {
+
+  Jurisdiction jurisdiction;
+
+  string public constant name = "SampleToken";
+  string public constant symbol = "TPL";
+  uint8 public constant decimals = 18;
+
+  function SampleToken(Jurisdiction _jurisdiction) public {
+    jurisdiction = _jurisdiction;
+    totalSupply_ = 0;
+  }
+
+  function mint(address _to, uint256 _amount)
+    onlyOwner
+    canMint
+    public
+    returns (bool)
+  {
+    require(jurisdiction.hasAttribute(_to, 'VALID'));
```

Comment:
This needs to be checked in `transfer` and `transferFrom` too.

Reply:
Thanks for catching this!
I added the overwrites, and tests specific to the token.